android/src/main/java/com/airbnb/android/react/maps/AirMapMarker.java

```diff
@@ -706,8 +789,18 @@ public void setImage(String uri) {
         String traffic = jsonObj.has("traffic") ? jsonObj.getString("traffic"): "0";
         String zoomLevel = jsonObj.has("zoomLevel") ? jsonObj.getString("zoomLevel"): "16";
         Boolean hasValetService = jsonObj.has("hasValetService") ? true: false;
-
-        String svgDataUrI = getCFSvg(hasValetService, Math.round(Float.parseFloat(level)), topOutline, bottomOutline, topInner, bottomInner, promotedMarkerArgs, depotType, Float.parseFloat(traffic), Integer.valueOf(zoomLevel));
+        JSONObject planned = jsonObj.getJSONObject("planned");
+        Boolean isPlanned = new Boolean(planned.getString("isPlanned"));
```

Comment:
need to add null check here for backward compatibility. In case an app version uses this without passing this new planned arg.